hawkbit-autoconfigure/src/main/java/org/eclipse/hawkbit/autoconfigure/security/SecurityManagedConfiguration.java

```diff
@@ -295,15 +299,52 @@ public FilterRegistrationBean eTagFilter() {
         // in the ShallowEtagHeaderFilter, just using the SH1 hash of the
         // artifact itself as 'ETag', because otherwise the file will be copied
         // in memory!
-        filterRegBean.setFilter(
-                new ExcludePathAwareShallowETagFilter("/rest/v1/softwaremodules/{smId}/artifacts/{artId}/download",
-                        "/{tenant}/controller/v1/{controllerId}/softwaremodules/{softwareModuleId}/artifacts/**",
-                        "/api/v1/downloadserver/**"));
+        filterRegBean.setFilter(new ExcludePathAwareShallowETagFilter("/UI/**",
```

Comment:
Double check the @Order() annotations in this class to have a clear structure. I saw that @Order(100) eTagFilter comes after @Order(300) ControllerSecurityConfigurationAdapter.